### ISSUE TITLE
`Basic`, `Advanced` and `On exit` tab should have the brave focus indicator for tabbed navigation 

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#25501]
## Description 

`Basic`, `Advanced` and `On exit` tab should have the brave focus indicator for tabbed navigation 

## Steps to Reproduce

   1. Clean profile 1.45.x
   2. Open brave://settings/clearBrowserData
   3. `Basic`, `Advanced` and `On exit` tab should have the brave focus indicator for tabbed navigation 
   

## Actual result:

![image](https://user-images.githubusercontent.com/38657976/191296699-23203aef-d4fc-4d78-9cf4-4801a5003ac2.png)

## Expected result:
`Basic`, `Advanced` and `On exit` tab should have the brave focus indicator for tabbed navigation 
Figma: https://www.figma.com/file/z9wmg2FCwuXx9FLbDo5avJ/Platform-UI---Brave-desktop?node-id=1980%3A30533

## Reproduces how often: 

Easy
## Brave version (brave://version info)

Brave | 1.45.59 Chromium: 106.0.5249.40 (Official Build) nightly (64-bit)
-- | --
Revision | fab1d91915d2722d6339aaa7f4e9ce44f1e9b103-refs/branch-heads/5249@{#442}
OS | Windows 10 Version 21H2 (Build 19044.2006)

## Version/Channel Information:

- Can you reproduce this issue with the current release? No
- Can you reproduce this issue with the beta channel? No
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/qa-team @aguscruiz @rebron 

please refer the thread for more details https://bravesoftware.slack.com/archives/C7VLGSR55/p1663684532118859?thread_ts=1663297942.152529&cid=C7VLGSR55

**Comment by rebron** [comment]
related issue: https://github.com/brave/brave-browser/issues/25498
